torch/export/_trace.py

```diff
@@ -1486,6 +1683,281 @@ def forward(self, *args, **kwargs):
     )
 
 
+@_log_export_wrapper
+@_disable_prexisiting_fake_mode
+def _export_for_training(
```

Comment:
This implementation has a lot of duplications with `_strict_export`. 
So I wonder what's the delta? can we reuse code?